doc/api/n-api.md

```diff
@@ -3269,9 +3269,11 @@ napi_status napi_create_async_work(napi_env env,
 - `[in] async_resource_name`: Identifier for the kind of resource that is
 being provided for diagnostic information exposed by the `async_hooks` API.
 - `[in] execute`: The native function which should be called to excute
-the logic asynchronously.
+the logic asynchronously. The given function is called from a worker pool
+thread and executes in parallel with the main event loop thread.
 - `[in] complete`: The native function which will be called when the
-asynchronous logic is comple or is cancelled.
+asynchronous logic is comple or is cancelled. The given function is called
```

Comment:
should this be completed and not comple?

Reply:
Good find, that was a typo in the original documentation. Fixed that as well.